src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/Authorisation.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.ons.ssdc.supporttool.endpoint;
+
+import static uk.gov.ons.ssdc.supporttool.model.entity.UserGroupAuthorisedActivityType.SUPER_USER;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import uk.gov.ons.ssdc.supporttool.model.entity.User;
+import uk.gov.ons.ssdc.supporttool.model.entity.UserGroupAuthorisedActivityType;
+import uk.gov.ons.ssdc.supporttool.model.entity.UserGroupMember;
+import uk.gov.ons.ssdc.supporttool.model.entity.UserGroupPermission;
+import uk.gov.ons.ssdc.supporttool.model.repository.UserRepository;
+import uk.gov.ons.ssdc.supporttool.security.UserIdentity;
+
+@RestController
+@RequestMapping(value = "/auth")
+public class Authorisation {
+  private final UserIdentity userIdentity;
+  private final UserRepository userRepository;
+
+  public Authorisation(UserIdentity userIdentity, UserRepository userRepository) {
+    this.userIdentity = userIdentity;
+    this.userRepository = userRepository;
+  }
+
+  @GetMapping
+  public Set<UserGroupAuthorisedActivityType> getAuthorisedActivities(
+      @RequestHeader(required = false, value = "x-goog-iap-jwt-assertion") String jwtToken,
+      @RequestParam(required = false, value = "surveyId") Optional<UUID> surveyId) {
+    String userEmail = userIdentity.getUserEmail(jwtToken);
+
+    // TODO: Remove this before releasing to production!
+    if (userEmail.equals("dummy@fake-email.com")) {
+      return Set.of(UserGroupAuthorisedActivityType.values());
+    }
+
+    Optional<User> userOpt = userRepository.findByEmail(userEmail);
+
+    if (!userOpt.isPresent()) {
+      throw new ResponseStatusException(HttpStatus.FORBIDDEN, "User not known to RM");
+    }
+
+    User user = userOpt.get();
+
+    Set<UserGroupAuthorisedActivityType> result = new HashSet<>();
+    for (UserGroupMember groupMember : user.getMemberOf()) {
```

Comment:
Not an issue now, but perhaps a candidate for future refactoring into something more readable